juicy-draggable.html

```diff
@@ -252,7 +252,13 @@
             var element = null;
             var handle = null;
             var that = this;
-            var root = this.createShadowRoot();
+            var root;
+            if (this.attachShadow) {
+                root = this.attachShadow({ mode: 'open' });
+            } else {
+                root = this.createShadowRoot();
```

Comment:
Wow, right. But this makes the whole `<template id="juicy-draggable-element">` redundant.